typescript/examples/vercel-ai/velodrome/README.md

```diff
@@ -0,0 +1,15 @@
+# Vercel AI with viem Example
```

Comment:
Why are we pushing this example? Does not include the velodrome plugin. I would remove this whole package, not needed